fastestimator/estimator.py

```diff
@@ -194,23 +206,21 @@ class System:
     def __init__(self,
                  mode: str,
                  global_step: int,
-                 batch_size: int,
                  num_devices: int,
                  log_steps: int,
                  total_epochs: int,
-                 total_steps: int,
                  epoch_idx: int,
                  batch_idx: int):
         self.mode = mode
         self.global_step = global_step
-        self.batch_size = batch_size
         self.num_devices = num_devices
         self.log_steps = log_steps
         self.total_epochs = total_epochs
-        self.total_steps = total_steps
         self.epoch_idx = epoch_idx
         self.batch_idx = batch_idx
         self.buffer = {}
+        self.loader = None
+        self.network = None
```

Comment:
why are we putting a reference to loader/network inside of System? this will create some circular dependency issues if we move system class to util folder

Reply:
moved system.

Reply:
question about why network is needed in system still open

Reply:
in case user wants to access some network property in Trace.

the goal is to enable the access and it's up to the user to use it.